website/docs/integrations/opentofu.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: OpenTofu Integration
+sidebar_position: 8
+sidebar_label: OpenTofu
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img width="250" align="right" src={useBaseUrl('/img/opentofu-icon.svg')} /> Atmos natively supports [OpenTofu](https://opentofu.org),
+similar to the way it supports [Terraform](/integrations/terraform). It's compatible with every version of `opentofu` and designed to
+work with multiple different versions of it concurrently, and can even work along side with [HashiCorp Terraform](/integrations/terraform). 
+
+Please see the complete configuration options for [Terraform](/integrations/terraform), as they are the same for OpenTofu. We'll focus
+only on what's different in this document, in order to utilize OpenTofu. Keep in mind that Atmos does not handle the downloading or installation
+of OpenTofu; it assumes that any required commands are already installed on your system.
+
+Additionally, if using Spacelift together with Atmos, make sure you review the [Spacelift Integration](/integrations/spacelift) to make any necessary changes.
+
+## CLI Configuration
+
+All of the default configuration settings to support OpenTofu are defined in the [Atmos CLI Configuration](/cli/configuration),
+but can also be overridden at any level of the [Stack](/core-concepts/stacks/#schema) configuration.
+
+To make OpenTofu the default command when running "terraform", modify [`atmos.yaml`](/cli/configuration) to configure the following global settings.
+
+```yaml
+components:
+  terraform:
+    # Use the `tofu` command when calling "terraform" in Atmos.
+    command: "/usr/bin/tofu"
+
+    # Optionally, specify a different path for OpenTofu components
+    base_path: "components/tofu"
+```
+
+:::important
+Atmos consistently utilizes the `terraform` keyword across all configurations, rather than `tofu` or `opentofu`.
```

Comment:
I was thinking of opening an issue for this, but thoughts on making an alias of `atmos tf` for `atmos terraform` . You can then think about it as either which may help in any confusion for folks who don't know atmos and want to run `tofu` out of the gate.

Reply:
Haha I didn't read the below and didn't know there was support for aliases via `atmos.yaml` config. Good to know. But I do think this question still stands as a utility to decrease confusion AND to make writing atmos commands more terse.

Reply:
Note, this is referring to keywords in configurations, not the commands used. There needs to be some canonical way to refer to stuff is run by Terraform and OpenTofu. For now, it's `terraform`. 

> But I do think this question still stands as a utility to decrease confusion AND to make writing atmos commands more terse.

So we're not going to support "canonical" short-terms, because there's no consensus on what's short enough, or what's ambiguous. However, that's why we added support for `aliases` which are configurable in `atmos.yaml` and can alias _any_ atmos command. 

For additional context and inspiration behind the `aliases` configuration, see #506.

- #506